### DISCUSSION TITLE
How to automate imports from GoCardless?

### DISCUSSION THREAD

**Discussion by wsw70** [firefly-iii/firefly-iii#9206] [Firefly III] [answered]
I set up the data importer following the [documentation ](https://docs.firefly-iii.org/tutorials/data-importer/gocardless/) and I can manually trigger the import of data from my accounts.

This is however a manual operation and I would like to automate it to run the import say once a day. Sorry if the question is obvious but the documentation does not mention it: how can I do that?

**Comment by digitlength** [comment, marked as answer]
> but the documentation does not mention it

https://docs.firefly-iii.org/how-to/data-importer/import/automated/

**Reply by wsw70** [reply]
Ah, see, it does 🫥🫣
Thank you very much and sorry for that!
